README.md

```diff
@@ -93,380 +93,54 @@
 
 ------------------------------------------------------------------------
 
-Ivy is both an ML transpiler and a framework, currently supporting JAX,
-TensorFlow, PyTorch, and Numpy.
+Ivy is an open-source machine learning framework that 
+enables you to: 
 
-Ivy unifies all ML frameworks 💥 enabling you not only to **write code
-that can be used with any of these frameworks as the backend**, but also
-to **convert 🔄 any function, model, or library written in any of them to
-your preferred framework!**
+- 🔥 **Autotune and optimize your model**: you can compress your model, 
+get the best cost, and more with Ivy. 
```

Comment:
I still think this is too vague, doesn't really explain what the autotuner is, seems to focus around compression when this is just a small step that is only applied in a subset of the transformations we do

Reply:
Fixed.